packages/supabase/src/mod.ts

```diff
@@ -16,9 +16,13 @@ export function supabaseAdapter<T>({ supabase, table }: { supabase: SupabaseClie
 
   return {
     read: async (id: string) => {
-      const { data, error } = await supabase.from<Session>(table).select('session').eq('id', id).single();
+      const { data, error } = await supabase.from(table).select('session').eq('id', id).limit(1).throwOnError().single();
```

Comment:
Thanks for the updates. But looks like there's bug with `throwOnError` so i think we don't need it here, see [this](https://github.com/supabase/supabase-js/issues/554) and [this](https://github.com/supabase/supabase/blob/ce6cf1e89eb67acf576129c0823ab362757cf902/apps/docs/docs/ref/javascript/release-notes.mdx#L257).

```suggestion
     const { data, error } = await supabase.from(table).select('session').eq('id', id).limit(1).single();
```